### PR TITLE
handbrake: update to 0.10.1.

### PR DESCRIPTION
**Pull request by lemmi**
please merge after x264, so handbrake can profit from asm optimizations

### CODE REVIEW

File: srcpkgs/handbrake/template [hunk 1]
```diff
@@ -17,8 +17,8 @@ short_desc="Multithreaded video transcoder"
 maintainer="Juan RP <xtraeme@gmail.com>"
 license="GPL-2"
 homepage="http://handbrake.fr/"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/HandBrake-${version}.tar.bz2"
-checksum=a91042da3cc08454844f7c171ee2fde4f0ed08176407fc0328d5118227ed4114
+distfiles="https://handbrake.fr/rotation.php?file=HandBrake-${version}.tar.bz2>HandBrake-0.10.1.tar.bz2"
```
Comment (Gottox) [inline srcpkgs/handbrake/template]: Use `$version` instead of a static version definition

**Comment by Gottox** [comment]
Looks okay to me. Waiting till #1166 is merged.

**Comment by xtraeme** [comment]
This one can be applied, thanks :+1:

**Comment by 38github** [comment]
Update: Version 0.10.5 is available from Handbrake. I tried to compile it for musl by just edit the template's version value and sha256 but the compile ended at:

```
libavresample/audio_mix_matrix.c:133:54: note: each undeclared identifier is reported only once for each function it appears in
libavresample/audio_mix_matrix.c:146:73: error: 'M_SQRT2' undeclared (first use in this function)
                 matrix[FRONT_CENTER][FRONT_CENTER] = center_mix_level * M_SQRT2;
                                                                         ^
make[1]: *** [Makefile:44: libavresample/audio_mix_matrix.o] Error 1
make[1]: Leaving directory '/builddir/HandBrake-0.10.5/build/contrib/ffmpeg/libav-v10.1'
make: *** [../contrib/ffmpeg/module.rules:3: contrib/ffmpeg/.stamp.build] Error 2
=> ERROR: handbrake-0.10.5_1: failed to run do_build() at line 39.
```

**Comment by lemmi** [comment]
@38github please open a new issue for that.

**Comment by ebfe** [comment]
@38github please don't #3987 :)

**Pull request merged**